Allow more verbose

jpereira opened the pull request:
perl -p -i -e 's/\t\@/\t\${Q}/' $(find . -name "all.mk" -print)

arr2036 commented:
The majority of the commands you removed the @ from were echo or printf, which is wrong.  We don't want:

```
echo DOXYGEN /path/to/doxygen
DOXYGEN /path/to/doxygen
```

in the output...

arr2036 commented on src/lib/all.mk:
```diff
@@ -19,7 +19,7 @@ src/freeradius-devel: | src/include/${1}
 src/include/${1}:
 	$${Q}[ -e $$@ ] || ln -sf $${top_srcdir}/src/lib/${1} $$@
-	@echo LN-SF src/lib/${1} $$@
+	${Q}echo LN-SF src/lib/${1} $$@
```
These all need to be escaped $${Q}

arr2036 replied:
But still, you're emitting the echo command... so not necessary to begin with

jpereira commented:
Got it @arr2036 . I updated and fixed some extra things like: `${Q}${Q}command....` and now we are using `@echo ...` only to print the first target message.
